.github/ISSUE_TEMPLATE/gsoc-project-component-ideas.md

```diff
@@ -1,6 +1,6 @@
 ---
-name: GSoC Project Ideas
-about: GSoC Project Ideas for Students
+name: Ideas for GSOC project components
+about: Ideas for GSOC project components from contributors
 title: "[GSoC]"
 labels: ideas
```

Comment:
```suggestion
title: ''
labels: ["ideas", "gsoc"]
```
Just a little tweak while you're at it 😄 I figured it's probably better to put the gsoc label rather than it being in the title.

Reply:
Sounds good!
Maybe we make it `gsoc2022_ideas`?